deploy/examples/create-external-cluster-resources.py

```diff
@@ -768,7 +771,7 @@ def get_ceph_dashboard_link(self):
 
     def create_rgw_admin_ops_user(self):
         cmd = ['radosgw-admin', 'user', 'create', '--uid', self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME, '--display-name',
-               'Rook RGW Admin Ops user', '--caps', 'buckets=*;users=*;usage=read;metadata=read;zone=read']
+               'Rook RGW Admin Ops user', '--caps', 'info=read;buckets=*;users=*;usage=read;metadata=read;zone=read']
```

Comment:
@leseb what about adding `info=read` for the admins ops user created by Rook to make it consistent

Reply:
That wouldn't hurt for sure but go-ceph does not support it yet. Perhaps we can do that in another PR once it's supported.

Reply:
As far as I understand go-ceph handles the` user_cap` struct as a generic struct containing `type` and `perm` as strings. So IMO no change is required from the go-ceph library and we can directly update the perms for `admin ops user`. On a different note if u need to include perms for `cephobjectstoreuser` then we need some changes for its Spec

Reply:
I meant handling the call to `info/` path.

Reply:
okay